tests/storybook/src/stories/blocksuite-editor.stories.tsx

```diff
@@ -37,7 +37,7 @@ DocEditor.loaders = [
       id: 'test-workspace-id',
       schema,
     });
-
+    workspace.doc.emit('sync', []);
```

Comment:
Does this sync event have a relation to this issue?

Reply:
Oh this is unrelated. I will split it into another PR